blog/2024/09-10-release-roadmap-10.10.0/index.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Release Roadmap for 10.10.0
+description: We are about to start the 10.10.0 release cycle; here is the roadmap. Please help us test!
+authors: joshuaboniface
+date: 2024-09-10
+slug: release-roadmap-10.10.0
+tags: [release, testing]
+---
+
+![Good heavens, look at the time!](./10.10.0.jpg)
```

Comment:
I would move the image to below the truncate, as the post preview is now quite large.

Reply:
It matches the call for developers, so I think it's OK. Part of the fun is that it's shown right under the title.

Reply:
Hmm yeah I can understand. Perhaps make it a bit smaller then. Looks quite big on the blog list page:

https://42ed1185.jellyfin-org.pages.dev/posts

Reply:
Shrunk it a decent bit, hopefully that looks better.